src/test/kotlin/RequestParserTest.kt

```diff
@@ -3,32 +3,36 @@ import request.RequestParser
 import request.Request
 
 class RequestParserTest {
-    @Test fun testReturnRequestWhenSimpleGetFromClient() {
+    @Test
+    fun `when parsing a simple_get from client then return Request`() {
         val parser = RequestParser()
         val clientRequest = "GET /simple_get HTTP/1.1\r\n" + "Connection: close\r\n" + "Host: 127.0.0.1:5000\r\n" +
                 "User-Agent: http.rb/4.3.0\r\n" + "Content-Length: 0"
         assertIs<Request>(parser.parse(clientRequest))
     }
 
-    @Test fun testReturnRequestWithSimpleGetRequestComponents() {
+    @Test
+    fun `when parsing a simple_get from client then Request contains http method and route`() {
         val parser = RequestParser()
         val clientRequest = "GET /simple_get HTTP/1.1\r\n" + "Connection: close\r\n" + "Host: 127.0.0.1:5000\r\n" +
                 "User-Agent: http.rb/4.3.0\r\n" + "Content-Length: 0"
 
-        val response = parser.parse(clientRequest)
+        val request = parser.parse(clientRequest)
```

Comment:
I missed this before! Good catch and rename.

Reply:
Thank you!